### PR TITLE
Adapt README.md

### PR DESCRIPTION
**Pull request by DunklesArchipel**
A more meaningful readme, reflecting better the content of this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,13 +31,73 @@ Alternatively, if you want to install the required packages into an existing, en
...
+The current state of the website can be downloaded and stored in a database.
 
+```python
+>>> from echemdb.data.cv.database import Database
+>>> db = Database()
 ```
-conda activate echemdb
-mkdocs serve
+
+Filtering the database for entries having specific properties, i.e., containing Pt as working electrode material, returns a new database.
```
Comment (saraedum) [inline README.md:51]:
```suggestion
Filtering the database for entries having specific properties, e.g., containing Pt as working electrode material, returns a new database.
```

File: README.md [hunk 2]
```diff
@@ -18,13 +31,73 @@ Alternatively, if you want to install the required packages into an existing, en
...
 ```
 
+Each entry has information about its source
+
+```python
+>>> entry.source # or entry['source']
+{'citation key': 'engstfeld_2018_polycrystalline_17743', 'curve': 1, 'url': 'https://doi.org/10.1002/chem.201803418', 'figure': '4b', 'version': 1}
+```
+
+Among other metadata, the entry also has information on the original figure properties (`entry.figure_description`) and the `entry.electrochemical_system` in general.
```
Comment (saraedum) [inline README.md:71]:
```suggestion
Among other metadata, the entry also has information on the original publication's figure properties (`entry.figure_description`) and the `entry.electrochemical_system` in general.
```

File: README.md [hunk 3]
```diff
@@ -18,13 +31,73 @@ Alternatively, if you want to install the required packages into an existing, en
...
+
+The data related to an entry can be returned as a [pandas](https://pandas.pydata.org/) dataframe (values are provided in SI units) and can be stored as CSV file (or any other format supported by pandas).
```
Comment (saraedum) [inline README.md:73]:
```suggestion
The data related to an entry can be returned as a [pandas](https://pandas.pydata.org/) dataframe (values are provided in SI units) and can be stored as a CSV file (or any other format supported by pandas).
```

File: README.md [hunk 4]
```diff
@@ -18,13 +31,73 @@ Alternatively, if you want to install the required packages into an existing, en
...
+
+```python
+>>> entry.df() 
+           t	        E	       j
+0	0.000000	-0.196962	0.043009
+1	0.011368	-0.196393	0.051408
+...
+# Custom or original figure axes units can be set specifically
```
Comment (saraedum) [inline README.md:81]:
```suggestion
# Custom or original figure axes' units can be requested explicitly 
```

File: README.md [hunk 5]
```diff
@@ -18,13 +31,73 @@ Alternatively, if you want to install the required packages into an existing, en
...
+The data related to an entry can be returned as a [pandas](https://pandas.pydata.org/) dataframe (values are provided in SI units) and can be stored as CSV file (or any other format supported by pandas).
+
+```python
+>>> entry.df() 
+           t	        E	       j
+0	0.000000	-0.196962	0.043009
+1	0.011368	-0.196393	0.051408
+...
+# Custom or original figure axes units can be set specifically 
+# entry.df(xunit='original', yunit='mA / m2')
```
Comment (saraedum) [inline README.md:82]: Why is this commented out?

File: README.md [hunk 6]
```diff
@@ -18,13 +31,73 @@ Alternatively, if you want to install the required packages into an existing, en
...
+>>> entry.df().to_csv('../testtesttest.csv', index=False)
+```
+
+The reference electrode to the potential axis `E` is provided in the `entry.data_description`
```
Comment (saraedum) [inline README.md:86]: Not sure but should it be "for" the potential axis?

File: README.md [hunk 7]
```diff
@@ -18,13 +31,73 @@ Alternatively, if you want to install the required packages into an existing, en
...
+>>> entry.data_description.axes.E.reference
+'RHE'
+```
+
+The data can be visualized in a plotly figure, with preferred axis units (default is SI):
+
+```python
+>>> entry.plot(xunit='V', yunit='original')
+```
+<!-- ![doc/images/readme_demo_plot.png](doc/images/readme_demo_plot.png)-->
```
Comment (saraedum) [inline README.md:97]:
```suggestion
```

**Review by saraedum — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (DunklesArchipel) [reply README.md:84]: I changed to "of the potential axis"

File: README.md (hunk 5 above)
Comment (DunklesArchipel) [reply README.md]: I thought everything would be too cluttered. I included it now since it is an interesting aspect.

**Pull request merged**